frontend/nginx/nginx.conf

```diff
@@ -0,0 +1,17 @@
+server {
+
+  listen 80;
+
+  location / {
+    root   /usr/share/nginx/html/public;
+    index  index.html index.htm;
```

Comment:
Is there a reason `root` is defined inside the `location`? I've seen `nginx.conf` files use both, but just wondering.

Reply:
It changes the root based on the location.